### DISCUSSION TITLE
Howto activate ipv6

### DISCUSSION THREAD

**Discussion by ftpn** [ej52/proxmox-scripts#24] [Q&A]
My provider uses ds lite so I need ipv6 working, how do I enable it?

**Comment by ej52** [comment]
@ftpn The default CT created by `setup.sh` only caters for IPv4 however, the alpine CT and NPM already support IPv6.

You would just need to setup IPv6 on the CT in order to get it working.
<img width="290" alt="Screenshot 2021-06-04 at 09 46 26" src="https://user-images.githubusercontent.com/6298706/120765715-c7179d80-c519-11eb-92d3-e456e2343eb2.png">
